fedrec/utilities/logger.py

```diff
@@ -4,12 +4,31 @@
 
 
 class BaseLogger(ABC):
+    """
+    This logger class performs the logging messages and
```

Comment:
It will be good if we can re-write it as:
```suggestion
    The `BaseLogger` performs the logging messages and
```